### DISCUSSION TITLE
Adding more info on the model-zoo page

### DISCUSSION THREAD

**Discussion by vikashg** [Project-MONAI/model-zoo#175] [Ideas]
Hi @zephyrie, 
So one of the discussions came up in the meeting last week, which I think is of interest to you and you can help us handle this. 
Different models may support different features. For example, 
1. Multi GPU training
2. Single GPU training
3. Inference
4. Inference but no training 
5. An associated MONAI Application Package is available from monai deploy app sdk
6. Supports torchscript
7. Supports yaml configuration
8. Support both yaml and json Configuration file 
9. monai release before 1.0 or after 1.0

I think we can define a glossry of icons for each major feature and place it next to the model name on the web page as is done in [restaurant menus](https://www.freepik.com/premium-vector/food-beverage-icon-set-fillio-black-icon-series_6169344.htm#query=restaurant%20menu%20icon&position=13&from_view=keyword) for indicating vegetarian and vegan options. 

Maybe in the lower right hand corner of each model box. 
What do you think ?

**Comment by vikashg** [comment]
Adding @wyli, @yiheng-wang-nv and @Nic-Ma  for further comments.

**Reply by yiheng-wang-nv** [reply]
Thanks for the suggestion @vikashg , add @zephyrie for further discussion.
